stubs/tqdm/METADATA.toml

```diff
@@ -0,0 +1,2 @@
+version = "4.62"
+python2 = true
```

Comment:
While tqdm supports Python 2, the stubs don't seem to be take Python 2 into account. In this case, just leave out the `python2` marker. Python 2 support in typeshed is not a priority anymore and hinders other improvements.